sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.1.0-beta.1 (Unreleased)
 
+### Bug Fixes
+
+- Fixed a memory leak issue in xml parser.
```

Comment:
nit: XML parser is internal, not relevant to the user, so instead use the verb rather than a noun.
Also nit: capitalize XML.
```suggestion
- Fixed a memory leak issue while parsing XML.
```